From dffeb6465bf3ba1833da64aae3e9455910b40339 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Fri, 20 Dec 2019 10:51:50 +0800
Subject: [PATCH] dt-bindings: rockchip-thermal: Support the RV1126 SoC
 compatible

Add a new compatible for thermal founding on RV1126 SoC.

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
Change-Id: I84e7ca521f4edce9516575bd54709326e62fc85c
---
 Documentation/devicetree/bindings/thermal/rockchip-thermal.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
index c08f66222c75..d0f7d1578343 100644
--- a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
@@ -3,6 +3,7 @@
 Required properties:
 - compatible : should be "rockchip,<name>-tsadc"
    "rockchip,rv1108-tsadc": found on RV1108 SoCs
+   "rockchip,rv1126-tsadc": found on RV1126 SoCs
    "rockchip,px30-tsadc":   found on PX30 SoCs
    "rockchip,rk1808-tsadc": found on RK1808 SoCs
    "rockchip,rk3228-tsadc": found on RK3228 SoCs
-- 
2.35.3

